projects/packages/my-jetpack/_inc/components/my-jetpack-screen/index.jsx

```diff
@@ -124,9 +128,15 @@ export default function MyJetpackScreen() {
 						<div id="jp-admin-notices" className="my-jetpack-jitm-card" />
 					</Col>
 				</Container>
+				<WelcomeBanner
+					onDismissBanner={ dismissWelcomeBanner }
+					dismissedWelcomeBanner={ window?.myJetpackInitialState?.dismissedWelcomeBanner === '1' }
```

Comment:
I think it's preferred to use `useSelector` way of accessing state, rather than window object variable. I see that window object is primarily used as a seed for the initial state of the reducer here:
https://github.com/Automattic/jetpack/blob/trunk/projects/packages/my-jetpack/_inc/state/store.js